zonal_datacube/geo_utils.py

```diff
@@ -0,0 +1,52 @@
+import math
+
+import numpy as np
+
+
+def get_hectare_area(lat: float, pixel_width: float) -> float:
+    """Calculate geodesic area for pixels of pixel_width * pixel_width
+    using WGS 1984 as spatial reference.
+    Pixel size various with latitude, which is it the only input parameter.
```

Comment:
Thanks for the doc, again! 💯 
Typo:
```suggestion
    Pixel size varies with latitude, which is the only input parameter.
```